awx/api/serializers.py

```diff
@@ -4380,37 +4381,65 @@ def validate(self, attrs):
 class WorkflowJobLaunchSerializer(BaseSerializer):
 
     can_start_without_user_input = serializers.BooleanField(read_only=True)
+    defaults = serializers.SerializerMethodField()
     variables_needed_to_start = serializers.ReadOnlyField()
     survey_enabled = serializers.SerializerMethodField()
     extra_vars = VerbatimField(required=False, write_only=True)
+    inventory = serializers.PrimaryKeyRelatedField(
+        queryset=Inventory.objects.all(),
```

Comment:
Do you want the `()` here?  Does this expect a callable instead like `Inventory.objects.all`?

Reply:
I think I was going by the docs here:

https://www.django-rest-framework.org/api-guide/relations/

I had used this pattern in some other serializers, it's when the field isn't an actual model field.

It should remain as an un-executed query, but it does still weird to type it out like this.